### PR TITLE
Adding Containers Dashbboard link to Menu.py

### PR DESCRIPTION
**Pull request by pavelzag**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit cd3d83b937a73fa45102c47e09cf3e656c83b215:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/containers/image_registry.py [hunk 1]
```diff
@@ -26,12 +27,29 @@ def __init__(self, host, provider):
         self.host = host
         self.provider = provider
 
+    def _on_detail_page(self):
+        return sel.is_displayed(
+            '//div//h1[contains(., "{} (Summary)")]'.format(self.host))
```
Comment (mfalesni) [inline cfme/containers/image_registry.py]: Now you can use `summary_title()` from `cfme.web_ui` to read the title and compare it to a string.

You can also use the `breadcrumbs()` function from the same module to check the breadcrumbs to verify you are on that page.

File: cfme/containers/container.py [hunk 2]
```diff
@@ -49,3 +51,12 @@ def navigate(self, detail=True):
     def click_element(self, *ident):
         self.load_details(refresh=True)
         return sel.click(details_page.infoblock.element(*ident))
+
```
Comment (psav) [inline cfme/containers/container.py]: These changes are unrelated and are already present in another PR.

**Comment by pavelzag** [comment]
Yea I have no idea how these things got mixed together.. I created different branches for each change. This confuses me

**Comment by psav** [comment]
Can this one also be closed as it's present in multiple other PRs?

**Comment by pavelzag** [comment]
Closed!
